### ISSUE TITLE
Bass shaker causing simhub crash

### ISSUE THREAD

**Issue by Pitrs1152** [SHWotever/SimHub#1712]
**Describe the bug**
Simhub can't allocate output connect to ButtKicker Gamer PRO. It gets this error 
ERROR - System.DllNotFoundException: Unable to load DLL 'fmodstudio': Invalid access to memory location.
In the Shakelt Bass Shakers tab there are no active outputs although buttkicker is selected.
After a while (aprox. 5min) it connects but it acts glitchy and stuttery for awhile and then crashes like it can't hold the allocation of the output. 
Only possible way to fix it is to reinstall the application but after computer restart the problematic behavior is back again. 

**Simhub version**
v9.4.9

**To Reproduce**
1. Install app, connect buttkicker
2. Restart PC

**Expected behavior**
Simhub not crashing and bass shaker working

**Screenshots**
![image](https://github.com/user-attachments/assets/7a00a377-3333-408c-b0a6-571ea15850fb)
![image](https://github.com/user-attachments/assets/710492d4-61ab-495f-97f7-5986af3a55b7)

**Log files**
[SimHub.txt](https://github.com/user-attachments/files/17202285/SimHub.txt)

**Additional context**

**Comment by SHWotever** [comment]
Hi !

Is it specific  to this sound card ? If you do enable the other does it connect ? (I understand it's not your bass shaker it's just meant as a test) 
I think the error listed is just a consequence of another issue during the connection.

I've built a version with more in depth logging (I don't expect to fix it, but hopefully it will give more insights about the true failure : https://simhubdash.com/unofficial/SimHub.9.04.10b1_FmodInit.zip could you give it a try and send me some fresh logs ?

**Comment by Pitrs1152** [comment]
This time I had to reset the PC twice to reproduce this, with the version you sent it never connected just kept showing connection error. 
I also tried my primary sound card and the error is the same there.  As no one else is reporting this I guess it is something very specific for my system ?
![image](https://github.com/user-attachments/assets/ba6991a3-44b7-4738-a691-36dfd1dfcf01)
[SimHub.txt](https://github.com/user-attachments/files/17203579/SimHub.txt)

**Comment by SHWotever** [comment]
Hi ! Thanks for the log, actually it's now properly seeing the initial error and reporting it. It's a progress as I can now spot the original failure, but not a solution indeed. 

I can see the failure during the FMod initialization. Usually when there is a "parasitic" behaviour, such as a bad sound driver it does fail later during the specific sound card "allocation". What is strange is that "random" behaviour, is there any audio related software you could be running or stopping later which could be a cause maybe ? I see only a second sound card, so it does not seems your computer is an "audio" crowded setup

Fmod is not extremely verbose about failures causes, so it can be a cat and mouse game. 

Just in case I did an FMod update in case this is something which got fixed in the FMod engine later : https://simhubdash.com/unofficial/SimHub.9.04.10b2_FmodInit.zip could you try it ?

**Comment by Pitrs1152** [comment]
I thought of that hijacking of a sound card by some other app too, but i always tested it when the windows freshly booted up. Although I have few apps that automatically start, when I tried scanning the ID of the device (buttkicker soundcard) trough process explorer to make sure, no app was using the device. 
This new version did not help, but I confirmed another symptom which I firstly thought was just a one time thing in the other versions but it happened again. After trying to launch Simhub and restarting my PC several times Simhub gets "stuck" and just crashes every time immediately after I start the app and nothing helps until I reinstall the app, there are no logs created just for a few seconds I see this in the task bar. I even tried disconnecting the sound card. It is very weird.
![image](https://github.com/user-attachments/assets/e7c30485-a7dd-443c-9f9f-264ebf548c45)
this is log from the last time I managed to run the app with last version you sent. 
[SimHub.txt](https://github.com/user-attachments/files/17216135/SimHub.txt)

**Comment by Pitrs1152** [comment]
I found what the issue was. It was something wrong with the system drivers, windows itself or the SSD just did not have enough space. Just this week this same issue started happening on random apps like Geforce Experience, Moza pit house even eset online scanner. I used all the tips from microsoft when getting app crashes, updated all drivers including bios but especially when I freed up 400GB of space on system SSD and everything started working as it should. I can only guess which of the steps helped but I think this was something that is extremly rare.
